package.bzl

```diff
@@ -32,10 +33,19 @@ def rules_postcss_dependencies():
         sha256 = "97e70364e9249702246c0e9444bccdc4b847bed1eb03c5a3ece4f83dfe6abc44",
     )
 
+    _include_if_not_defined(
+        git_repository,
+        name = "io_bazel_stardoc",
+        remote = "https://github.com/bazelbuild/stardoc.git",
+        commit = "247c2097e7346778ac8d03de5a4770d6b9890dc5",
+    )
+
     # NodeJS rules.
     _include_if_not_defined(
         http_archive,
         name = "build_bazel_rules_nodejs",
+        # Un-dummy-ify skylib loading so we can dep on bzl_library targets.
```

Comment:
Can you include more detail in this comment about why this is necessary?

Reply:
Done.